x-pack/plugins/maps/public/connected_components/layer_panel/layer_settings/layer_settings.tsx

```diff
@@ -66,6 +72,25 @@ export function LayerSettings(props: Props) {
     }
   };
 
+  const renderIsFittable = () => {
+    if (!props.supportsFitToBounds) {
+      return null;
+    }
+    return (
+      <EuiFormRow display="columnCompressedSwitch">
+        <EuiSwitch
+          label={i18n.translate('xpack.maps.layerPanel.settingsPanel.fittableFlag', {
+            defaultMessage: `Include bounds when fitting on all data`,
```

Comment:
cc @gchaps

I think the copy on this option needs a little work. How about "Include layer in fit to data bounds computation".

It think it would also be useful to include a `?` tooltip to provide a deeper explanation of this option and when its useful. Maybe something like the below

"Fit to data bounds adjusts your map extent to show all of your data. Sometimes layers provide reference data and should not be included in the fit to data bounds computation. Use this option to exclude a layer from fit to data bounds computation."

Reply:
made some slight mods to make it shorter. 👍 👎 ?

Reply:
Is "fitting on" correct? Simply "fitting" sounds better to me, but I don't know the context.

Reply:
@gchaps I'd say the "fitting on" is more correct. Rather than "fitting the data" (~ which would apply adjusting the data somehow), it is the reserve. The map "fits _on_ the data", animating to the extent of all the data on the map. This PR allows layers to opt-out of this computation.